runtime/thunk/thunk.ts

```diff
@@ -142,53 +143,57 @@ async function doFindAndExecutePackFunction<T extends FormulaSpecification>({
           break;
         case MetadataFormulaType.PropertyAutocomplete:
           const syncTable = syncTables?.find(table => table.name === formulaSpec.syncTableName);
-          const autocompleteFn = ensureExists(syncTable?.propertyAutocomplete);
-          const propertyValues = {};
+          const autocompleteFormula = syncTable?.namedAutocompletes?.[formulaSpec.autocompleteName];
```

Comment:
where would `autocompleteName` come from? specifically:
- maybe some helper method of autocomplete name is needed somewhere.
- is it possible that an identity level autocomplete function would respond to a static property? (in which case we may have two autocomplete functions responding to a single property)

Reply:
The name here is the same string value that gets stored into the schema itself, which would be `propertyName as AutocompleteReference` by the convention of what `makeSyncTable()` does.

> is it possible that an identity level autocomplete function would respond to a static property? (in which case we may have two autocomplete functions responding to a single property)

A person could technically set a static property up with `autocomplete: AutocompleteType.Dynamic` to point it at the sync table's general dynamic autocomplete function instead of a property-specific one, even for a non-dynamic sync table.

Reply:
For completeness of discussion, there's another option that's different from what the code has now:

Instead of putting the property name itself into the `autocomplete` value in the schema, we could put some sentinel value like `__coda_static__` and then use the property name at runtime to map back to a particular autocomplete formula from `namedAutocompletes`

My motivation for filling in a string property name value during `makeSyncTable()` instead of just a marker like `__coda_static__` and resolving the property name runtime later was mostly to eliminate uncertainty and/or annoyance related to schema normalization: if I set up `syncTable.namedAutocomplete[foo] = ...` and `property.Foo.autocomplete = "foo"` at the same moment in `makeSyncTable` then there's no possibility for inconsistency about whether one name is normalized and the other isn't